### DISCUSSION TITLE
Maker Bundle creates controller without view template

### DISCUSSION THREAD

**Discussion by anton-os-dev** [symfony/symfony#51623] [Q&A]
Steps to reproduce:
```
symfony new folder --version="6.3.*" --webapp
composer require maker --dev
php bin/console make:controller DefaultController
```

Result:
`class DefaultController extends AbstractController
{
    #[Route('/default', name: 'app_default')]
    public function index(): JsonResponse
    {
        return $this->json([
            'message' => 'Welcome to your new controller!',
            'path' => 'src/Controller/DefaultController.php',
        ]);
    }
}`
twig template has not been created

Expected result: index method must return render() function, and twig template must be created

**Comment by ghost** [comment]
Did you install Twig? I had this issue but then realised I was using a skeleton version of symfony, where twig dependancy isn't installed.
